community/examples/tutorial-starccm-slurm.yaml

```diff
@@ -0,0 +1,124 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: starccm-on-slurm
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: starccm-slurm
+  region: us-central1
+  zone: us-central1-c
+
+# Documentation for each of the modules used below can be found at
+# https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/README.md
+
+deployment_groups:
+- group: primary
+  modules:
+  # Source is an embedded resource, denoted by "resources/*" without ./, ../, /
+  # as a prefix. To refer to a local resource, prefix with ./, ../ or /
+  # Example - ./resources/network/vpc
+  - id: network1
+    source: modules/network/vpc
+
+  - id: homefs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      local_mount: /home
+
+  - id: login-script
+    source: modules/scripts/startup-script
+    kind: terraform
+    settings:
+      runners:
+      - type: shell
+        content: |
+          #!/bin/bash
+          ssh-keygen -q -t rsa -N '' -f ~/.ssh/id_rsa <<<y >/dev/null 2>&1
+          cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+          echo -e "Host *\n StrictHostKeyChecking no" >> ~/.ssh/config
+          chmod 644 ~/.ssh/authorized_keys
+          chmod 644 ~/.ssh/config
+        destination: /tmp/permissive-ssh.sh
+
+  - id: compute-script
+    source: modules/scripts/startup-script
+    kind: terraform
+    settings:
+      runners:
+      - type: shell
+        content: |
+          #!/bin/bash
+          google_mpi_tuning --hpcthroughput
+          google_mpi_tuning --nomitigation
+          yum -y install libExt libXext.x86_64 nmap
```

Comment:
Would you be able to add a comment describing why these libraries are needed?

Reply:
It looks like the current version of the Slurm image works without the install. I will remove.